aioseop_class.php

```diff
@@ -4382,10 +4382,10 @@ function wp_head() {
 		if ( ! $this->is_page_included() ) {
 
 			$aioseop_robots_meta = new AIOSEOP_Robots_Meta();
-			$robots_meta         = $aioseop_robots_meta->get_robots_meta();
+			$robots_meta         = $aioseop_robots_meta->get_robots_meta_tag();
 
 			if ( ! empty( $robots_meta ) ) {
-				echo sprintf( '<meta name="robots" content="%s"', esc_attr( $robots_meta ) ) . " />\n";
+				echo $robots_meta;
```

Comment:
I'm surprised this didn't get flagged as needing an `esc_*` function (even though there is one in `AIOSEOP_Robots_Meta::get_robots_meta_tag_helper()`). Not sure if this should be changed or not.

Reply:
No need since it's already being escaped in `AIOSEOP_Robots_Meta::get_robots_meta_tag_helper()`. Otherwise we're just doing it twice.